<!DOCTYPE html>
<html>

<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">


    <script>
        var element = document.querySelector('[rel="stylesheet"]');
        var stylesheet = element.sheet || element.styleSheet;
        console.log(stylesheet);
        stylesheet.disabled = true;
    </script>

    <link rel="stylesheet" href="css/goodload.css">

    <!-- Habilitar nomes al app per veure les funcions Jquery -->
    <script src="dist/js/vendor.js"></script>

    <title>BIO IQS</title>

    <!-- Bootstrap core CSS -->
    <link href="css/bootstrapmin.css" rel="stylesheet">

    <script>
        $(function () {
            $('navigationbar').load('/templates/header.html');
            $('footer').load('/templates/footer.html');
        });
    </script>

    <!--    Fonts googlefonts-->
    <link href="https://fonts.googleapis.com/css?family=Permanent+Marker" rel="stylesheet">

    <!-- Custom fonts for this template -->
    <link href="css/allmin.css" rel="stylesheet"> <!-- canvio all.min per allmin-->
    <link rel="stylesheet" href="css/simplelineicons.css">     <!-- CANVI-->
    <link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet">
    <link href="https://fonts.googleapis.com/css?family=Catamaran:100,200,300,400,500,600,700,800,900" rel="stylesheet">
    <link href="https://fonts.googleapis.com/css?family=Muli" rel="stylesheet">


    <link rel="stylesheet" href="css/new-age.css">

    <link rel="stylesheet" href="css/adaptwiki.css">


    <!--Added MathJax hack to make inline equations work-->
    <script type="text/x-mathjax-config">
    MathJax.Hub.Config({
    tex2jax: {
    inlineMath: [ ['$','$'], ["\\(","\\)"] ],
    processEscapes: true
  }
});
</script>
</head>

<body id="page-top">
    <div id="igem-navbar-placeholder">
        <!-- p>Placeholder for iGEM Navbar</p> -->
    </div> <!-- TREIEM AIXO PQ SENSE ESTIL ES PERD EL FORMAT I NO SERVEIX DE RES -->
    <navigationbar></navigationbar>
    
    <button id="myBtn" class="js-scroll-trigger" title="Go to top">Top</button>

    <header class="masthead little">
        <div class="container h-100">
            <div class="row h-100">
                <div class="col-lg-7 my-auto">
                    <div class="header-content">
                        <h1 class="mb-5">Wet Lab | Expression</h1>
                        <a href="#first-steps" class="btn btn-outline btn-xl js-scroll-trigger">Have a look!</a>
                    </div>
                </div>
            </div>
        </div>
    </header>
    
    <button onclick="topFunction()" id="myBtn" class="js-scroll-trigger" title="Go to top">Top</button>

    <section class="first-steps text-center" id="first-steps">
        <div class="container">
            <div class="row">
                <div class="col-md-9 mx-auto">
                    <h2 class="section-heading orange">Before starting</h2>
                    <div class="col-md-auto">
                       
                        <h4 class="book orange block-sept comas"><i>The HLA-DQ is a transmembrane protein that is involved in celiac disease. It is formed by two subunits, &alpha; and &beta;, each one located in a different loci from the human genome. The extracellular domain of each chain form a binding cleft that allows the interaction of different peptides derived from antigens (in this case, gluten derived peptides). These peptides usually have between 10 and 30 residues and this complex is presented to the for recognition by CD4 T-cells. </i></h4>
                    </div>
                </div>
                <div class="col-md-12 mx-auto block-sept">
                    <a class="js-scroll-trigger a-arrow" href="#int-human"><span class="arrow down"></span></a>
                </div>
            </div>
        </div>
    </section>
<!-- okay -->
    <section class="st-description block-desc b-orange-intense" id="int-human">
        <div class="container">
            <div class="row">
                <div class="col-md-12 mx-auto">
                    <div class="row">
                        <div class="col-md-1">
                        </div>
                        <div class="col-md-8 left">
                            <div class="row block-sept vertical-lines">
                                <div class="col-md-12">
                                   <h4 class="bold white"><u>Objectives</u></h4>
                                    <p class="bold white">Our goal was to obtain the extracellular domains of each subunit through a genome fishing assay or by a de novo design of a synthetic construct. These constructs would then be cloned in a bacterial host (<i>Escherichia coli</i>) or a yeast host (<i>Pichia pastoris</i>) and protein expression would be studied.</p>
                                </div>
                            </div>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-md-2">
                        </div>
                        <div class="col-md-8 left">
                            <div class="row block-sept vertical-lines">
                                <div class="col-md-12">
                                    <h4 class="bold white"><u>Important notes for protein expression</u></h4>
                                    <p class="bold white">1- In line with this strategy, there were some aspects that needed to be addressed when expressing the proteins. Firstly, the elimination of the transmembrane domain could increase the expression yield and the solubility since the resulting protein is smaller. However, these changes could also affect negatively to the interaccion affinity between the two chains. Additionally, high expression vectors, such as those containing the T7 promoter, could lead to the formation of inclusion bodies and therefore, refolding protocols would be necessary to purify the interested protein.</p>
                                    
                                    <p class="bold white">2- On the other hand, an analysis of posttranslational modifications (PTM) was performed to see whether the expression in <i>E.coli</i> would be sufficient or a higher evolved organism would be needed. We found two N-glycosylations (Glc-NAc) in the α-chain and one in the ß-chain, all of them placed in asparagine residues (αN103, αN143 adn ßN51). However, these PTM were found far away from the interaction pocket and therefore, we concluded that they would not affect the interaction between the gluten peptide and the HLA-DQ protein.</p>
                                    
                                    <p class="bold white">Based on the PTM analysis, simple systems such as <i>E.coli</i> could be used to express this protein. However, we also found the presence of disulfide bonds in both chains that could potentially affect the interaction pocket. Since <i>E.coli</i> is unable to make disulphide bonds, we decided to include synthetic constructs to express in the yeast host <i>Pichia pastoris</i>.</p>
                                </div>
                            </div>
                        </div>
                    </div>
                    
                </div>
            </div>
        </div>
    </section>
<!-- okay -->
   
   <section class="st-description block-desc" id="cl-description">
        <div class="container">
            <div class="row">
                
            <div class="col-md-12 mx-auto">
                <div class="row justify-content-center block-sept">
                    <div class="col-md-8 block-sept">
                        <h4 class="book orange-intense text-center">Construct design</h4>
                        <p class="book orange text-center">1- Construct #2a: Policistronic_EColi_CodonOptimisedK12_NdeISacI. This is a gBlock fragment that contains both alpha and beta chains in the same construct separated by an intergenic sequence with an additional RBS. This gene fragment is flanked by two restriction enzymes NdeI and SalI to enable cloning by restriction-ligation to a vector. After cloning to the vector, it can purified by a Strep-tactin purification column.</p>
                        <div class="image pers">
                         <img class="img-f" src="images/imagen_1.png">
                        </div>
                    </div>
                </div>
                <div class="row justify-content-center block-sept">
                    <div class="col-md-8 block-sept">
                        <p class="book orange text-center">2- Construct #2b: Policistronic_2tags_EColi_CodonOptimisedK12_NdeISacI. This construct was ordered as a synthetic gene. It is basically the same as #2a but it incorporates an extra Strep-tag at the end of the alpha chain.
                        </p>
                        <div class="image pers">
                         <img class="img-f" src="images/imagen_2.png">
                        </div>
                    </div>
                </div>
                <div class="row justify-content-center block-sept">
                    <div class="col-md-8 block-sept">
                        <p class="book orange text-center">3- Construct #3a: LZipper_EColi_CodonOptimisedK12_NdeISacI. This is a gBlock fragment derived from #2a that incorporates the Fos and Jun region at the end of each chain through a flexible linker (VDGGGGG). These regions are known to form the Leucine Zippers (LZ), a protein motif that contains a periodic repetition of a leucine residue at every seventh position. This heptad repeat forms a stable α-helical conformation that facilitates dimerisation and increases the stability of the oligomerized proteins.
                        </p>
                        <div class="image pers">
                         <img class="img-f" src="images/imagen_3.png">
                        </div>
                    </div>
                </div>
                <div class="row justify-content-center block-sept">
                    <div class="col-md-8 block-sept">
                        <p class="book orange text-center">4- Construct #3b: LZipper_2tags_EColi_CodonOptimisedK12_NdeISacI. Very similar to #3a, this gBlock fragment has, in addition to the LZ, a Strep-Tag at the end of each chain to facilitate the purification.
                        </p>
                        <div class="image pers">
                         <img class="img-f" src="images/imagen_4.png">
                        </div>
                    </div>
                </div>
                <div class="row justify-content-center block-sept">
                    <div class="col-md-8 block-sept">
                        <p class="book orange text-center">5- Construct #4b. Yeast_2A_2Tags_EcoRIXbaI_PichiaOptimised: This construct was ordered as a synthetic gene. It contains both alpha and beta chain separated by a sequence that encodes for the 2A cleavage site which is a short peptide sequence that has an auto-cleavage activity. This construct would result in the expression of both chains separately. The restriction sites of EcoRI and XbaI were added at the end of each chain to enable cloning into yeast vectors. We also added a His-tag to enable protein purification.
                        </p>
                        <div class="image pers">
                         <img class="img-f" src="images/imagen_5.png">
                        </div>
                    </div>
                </div>
                <div class="row justify-content-center block-sept">
                    <div class="col-md-8 block-sept">
                        <p class="book orange text-center">6- Construct #6d. Yeast_LZ_2A_2Tags_EcoRIXbaI_PichiaOptimised: This construct is a combination of construct #4b and #3a. It has been designed for yeast expression with addition of the Fos and Jun motifs to increase the protein stability.
                        </p>
                        <div class="image pers">
                         <img class="img-f" src="images/imagen_6.png">
                        </div>
                    </div>
                </div>
                <div class="row justify-content-center block-sept">
                    <div class="col-md-8 block-sept">
                        <p class="book orange text-center">7- Construct #7. Linker_EcoRINdeI_XbaISalI_Pichia_Ecoli_Optimized: This construct contains the extracellular domain of the alpha and the beta chain connected with a flexible linker. This construct was codon optimized so that it could be used in both <i>E.coli</i> and <i>Pichia pastoris</i>. 
                        </p>
                        <div class="image pers">
                         <img class="img-f" src="images/imagen_7.png">
                        </div>
                    </div>
                </div>
            </div>
            </div>
        </div>
    </section>
<!-- okay -->
  <section class="st-description block-sept" id="st-description">
        <div class="container">
            <div class="row">
                <p class="orange-medium book">The intended vectors to be used for cloning purposes are described below:</p>
            </div>

            <div class="col-md-12">
                <div class="row justify-content-center">
                    <div class="col-md-8">
                        <ul>
                            <li class="orange"> pET22b(+)—BsPdaC. This is a common pET22b(+) vector that has a 835bp gene between the restriction sites NdeI and SacI/SalI. Behind the SalI site there is a sequence that encodes for a Strep-Tag. This vector has been used to clone the constructs for <i>E.coli</i> expression.
                            </li>
                            <li class="orange">pPICZα: This is a 3.6 kb vector used to express recombinant proteins fused to N-terminal peptide encoding the Saccharomyces cerevisiae factor secretion signal. This vector allow a high-level, methanol inducible expression of the gene of interest in <i>Pichia</i> strain X-33. This vector contains Zeocin resistance gene for selection in both <i>E.coli</i> and <i>Pichia</i> and a C-terminal peptide containing the c-myc epitope and a polyhistidine (6xHis) tag for detection and purification of a recombinant fusion protein.</li>
                            <li class="orange">pGAPZα: This is a 3.1 kb vector designed for a high-level, constitutive expression in <i>Pichia pastoris</i>. This vector contains glyceraldehyde-3-phosphate dehydrogenase (GAP) promoter instead of methanol-regulated AOX1 promoter. It also contains Zeocin resistance gene, a C-terminal peptide containing the c-myc epitope and a polyhistidine (6xHis) tag and a Saccharomyces cerevisiae -factor secretion signal.</li>
                        </ul>
                    </div>

                </div>
            </div>

        </div>
    </section>
    
    <!-- okay -->
    <section class="st-description block-desc" id="cl-description">
        <div class="container">
            <div class="row">
                
            <div class="col-md-12 mx-auto">
                <div class="row justify-content-center block-sept">
                    <div class="col-md-8 block-sept">
                        <h4 class="book orange-intense text-center"><i>E.coli</i> constructs</h4>
                        <p class="book orange text-center"><u>Week 1 (Aug 20th-24th)</u>
                        Transformed all gene constructs (2b, 4b, 6d and 7) successfully and prepared a miniprep from them as well as the vector pET22b-BsPdaC. All gBlocks (2a, 3a, 3b) and construct 2b were digested with NdeI and SalI. A ligation was performed for each gene into the vector pet22b and transformed in <i>E.coli</i>. Results of the experiment showed high background, and colony PCR of 13 samples and gel analysis resulted in undesired band corresponding to the undigested vector carrying the BsPdaC gene (Figure 1).</p>
                        <div class="image pers">
                         <img class="img-f" src="images/imagen_8.png">
                        </div>
                        <p></p>
                        <p class="book orange text-center"><u>Week 2 (Aug 27th-31st)</u>
                        Repeated ligation and transformation of the same constructs (2a, 2b, 3a, 3b) but no colonies arose.</p>
                        <p></p>
                        <p class="book orange text-center"><u>Week 3 (Sep 3rd-7th)</u>
                        Performed restriction-ligation-transformation of construct 2b and 7 with a different restriction buffer and quantifying DNA before ligation. 21 colonies were observed for pet22b-2b and 7 colonies were observed for pet22b-7. 11 colonies were screened by PCR but results were inconclusive due to technical error.</p>
                        <p></p>
                        <p class="book orange text-center"><u>Week 4 (Sep 12th-15th)</u>
                        Unexpected event occurred. Several petri dishes kept in the fridge froze (including transformations from ligations) and lost most of the potential positive colonies. Repeated colony PCR for 4 viable samples. Gel analysis showed a band around 1300bp in sample nº8 that would correspond to the construct 2b (Figure 2a). Made a miniprep and analytical restriction of the positive colony, which showed promising results (Figure 2b). Transformed the plasmid into BL21 strain <i>E.coli</i>.</p>
                        <div class="image pers">
                         <img class="img-f" src="images/imagen_9.png">
                        </div>
                        <p></p>
                        <p class="book orange text-center"><u>Week 5 (Sep 17th- 21th)</u>
                        The positive colony was sequenced and resulted to be correctly cloned into the vector. A first test of expression was performed by boiling the cell pellet in a SDS-PAGE buffer and then run them in a acrylamide gel, but results were not conclusive. Besides that, a Gibson assembly reaction was performed using the constructs and #3b and the digested pET22b vector. Electroporated cells resulted in the growth of several colonies but colony PCR and analytical digestion showed a band with a lower molecular weight than expected (Figure 3).</p>
                        <div class="image pers">
                         <img class="img-f" src="images/imagen_10.png">
                        </div>
                        <p></p>
                        <p class="book orange text-center"><u>Week 6-7 (Sep 24th- Oct 5th)</u>
                        A second and third attempt of HLA-DQ expression were performed. Briefly, cultures were started at an initial OD600 of 0.1 and induced with IPTG when reached 0.3. Cultures were then incubated at 37ºC 220rpm for 6h and then harvested. Purification with a Strep-tactin column was performed after sonication but a very small peak was observed. Pellets were solubilized with 8M urea and samples were analysed with an acrylamide gel. Results show the presence of two intense bands at the solubilized fraction but the molecular weight were higher than expected (Figure 4).</p>
                        <div class="image pers">
                         <img class="img-f" src="images/imagen_11.png">
                        </div>
                        <p></p>
                        <div class="image pers">
                         <img class="img-f" src="images/imagen_12.png">
                        </div>
                    </div>
                </div>
            </div>
            </div>
        </div>
    </section>
    <!-- okay -->
    
    <section class="st-description block-desc" id="cl-description">
        <div class="container">
            <div class="row">
                
            <div class="col-md-12 mx-auto">
                <div class="row justify-content-center block-sept">
                    <div class="col-md-8 block-sept">
                        <h4 class="book orange-intense text-center"><i>P. pastoris</i> constructs</h4>
                        <p class="book orange text-center">All <i>Pichia</i> derived constructs and were obtained and digested correctly. Different ligation assays were performed but only one resulted to be successful. We obtained potential positive colonies in the zeocin resistant petri plates. However, analytical digestions of those colonies indicated that the presence of the band was outside the detection limit in agarose gel 1%.</p>
                    </div>
                </div>
            </div>
            </div>
        </div>
    </section>
	
    <footer></footer>


    <script src="/js/wiki.js"></script> <!-- calling all minified scripts -->


</body>

</html>


